### DISCUSSION TITLE
Bluetooth Serial

### DISCUSSION THREAD

**Discussion by Niassamond1** [Navideck/universal_ble#39] [General]
Hello, i was wondering if its possible to do serial bluetooth with this library. I've tried but couldnt do it.

**Comment by rohitsangwan01** [comment]
@Niassamond1 No Serial Bluetooth not supported

**Reply by fotiDim** [reply]
Isn't Serial Bluetooth only available in Bluetooth Classic?

**Reply by fotiDim** [reply]
@Niassamond1 do refer to Bluetooth Serial Port Profile (SPP) or something else which goes over BLE?
